core/arch/arm/kernel/delay.c

```diff
@@ -44,3 +48,37 @@ void mdelay(uint32_t ms)
 {
 	udelay(1000 * ms);
 }
+
+uint64_t utimeout_init(uint32_t us)
```

Comment:
s/utimeout_init/timeout_init_us()/ perhaps? We are not obliged to mimic `udelay()`.

Reply:
Ok, `timeout_init_us()` is fine to me.